### ISSUE TITLE
feature request: add live tv epg

### ISSUE THREAD

**Issue by basco** [styxit/HTPC-Manager#350]
is it possible to extend the live tv features? a nice epg and show program information etc.

**Comment by Hellowlol** [comment]
Isnt that possible now? 

There is a getepg https://github.com/styxit/HTPC-Manager/blob/master/modules/tvheadend.py#L32

**Comment by basco** [comment]
how do i access this from the htpc manager gui?

**Comment by Hellowlol** [comment]
Seems like its just epg for that channel :(. You click the channel icon :)

**Comment by Morphy99** [comment]
+1 for a full epg

**Comment by Hellowlol** [comment]
Isn't that module broken?

**Comment by Morphy99** [comment]
Not sure exactly but v 15 has some api fixes:
Improved PVR, EPG, and OSD frontend GUI in Kodi and extended the PVR API to support more features and functions
Improved PVR API and database adding many new EPG and Channel properties

**Comment by Hellowlol** [comment]
I was talking about the tvheadend module in htpc manager

**Comment by spicypixel-** [comment]
Doesn't seem to be in my module list, was it removed?

**Comment by Hellowlol** [comment]
@spicypixel- Sure your not using my fork? I have removed it from mine as its broken after a api update.

**Comment by spicypixel-** [comment]
Ah yes it's from your fork, guess it's best disabled then.

**Comment by Hellowlol** [comment]
Yeah, i have pinged the original author but he didnt respond. I dont have a tvcard and the api docs suck (atlest last time i checked). I cant fix it :(. I havnt removed the code, its just disabled and im hoping that some user will send a pr to fix it :D
